terraform/main.tf

```diff
@@ -441,7 +441,7 @@ resource "aws_s3_bucket_notification" "grant_prepared_data" {
   lambda_function {
     lambda_function_arn = module.PersistGrantsGovXMLDB.lambda_function_arn
     events              = ["s3:ObjectCreated:*"]
-    filter_suffix       = "/grants.gov/v2.xml"
+    filter_suffix       = "/grants.gov/v2.OpportunitySynopsisDetail_1_0.xml"
```

Comment:
For initial deploy, we want to add this as a new `lambda_function` alongside this existing one (to ensure any in-flight events get handled)